liamg/aminal
# implemented 'area scrolling'

## Description

Implemented scrolling of a group of lines within set margins
Fixes # 183

## Type of change


## How Has This Been Tested?

* Start aminal.
* Run htop
* Enter F4 to use the filter feature
* Enter a search string which will greatly limit the number of processes to display (I use `snap` or `aminal`).
* Effects stated in # 183 should no longer appear.

**Test Configuration**:
* OS: Ubuntu
* OS version: 18.04
* Go version: 1.10.4
